SwrveSDK/src/main/java/com/swrve/sdk/messaging/SwrveMessage.java

```diff
@@ -128,21 +123,16 @@ protected void setCacheDir(File cacheDir) {
     /**
      * @return the related campaign.
      */
-    public SwrveCampaign getCampaign() {
+    public SwrveInAppCampaign getCampaign() {
         return campaign;
     }
 
-    protected void setCampaign(SwrveCampaign campaign) {
+    protected void setCampaign(SwrveInAppCampaign campaign) {
         this.campaign = campaign;
     }
 
     private SwrveMessageFormat createMessageFormat(SwrveMessage swrveMessage, JSONObject messageFormatData) throws JSONException {
-        int defaultBackgroundColor = campaignManager.getConfig().getDefaultBackgroundColor();
-        return new SwrveMessageFormat(swrveMessage, messageFormatData, defaultBackgroundColor);
-    }
-
-    public ISwrveCampaignManager getCampaignManager() {
-        return campaignManager;
+        return new SwrveMessageFormat(swrveMessage, messageFormatData);
```

Comment:
No need for this method now. Just call new SwrveMessageFormat wherever its needed.